### PR TITLE
docs: corrected documentation links (#2045)

### PR DESCRIPTION
**Pull request by sagarshah1983**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

File: docs/versioned_docs/version-v1.7/jwks.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ GEs=
 ```
 
 ORY Hydra offers an API for generating and managing JWKs, the
-[JSON Web Keys API](https://www.ory.sh/docs/api/hydra). When using persistent
+[JSON Web Keys API](https://www.ory.sh/hydra/docs/reference/api/). When using persistent
```
Comment (aeneasr) [inline docs/versioned_docs/version-v1.7/jwks.md:43]:
```suggestion
[JSON Web Keys API](./reference/api). When using persistent
```

File: docs/versioned_docs/version-v1.7/jwks.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ JWKs are well supported amongst all languages. This endpoint helps you managing
 certificates, private, public and symmetric keys. It is important to never
 transport keys over insecure channels such as http.
 
-The [REST API Documentation](https://www.ory.sh/docs/api/hydra/) will give you
+The [REST API Documentation](https://www.ory.sh/hydra/docs/reference/api/) will give you
```
Comment (aeneasr) [inline docs/versioned_docs/version-v1.7/jwks.md:52]:
```suggestion
The [REST API Documentation](./reference/api) will give you
```

**Review by aeneasr — COMMENTED** [review]
Looking good! You can also use relative links as pointed out in the comments :)

Could you please also apply these changes to this document: https://github.com/ory/hydra/blob/master/docs/docs/jwks.md ? :)

**Comment by sagarshah1983** [comment]
@aeneasr Thanks for your suggestions. I have incorporated those.

**Comment by sagarshah1983** [comment]
@aeneasr -  Is anything pending on this PR. Please let me know. Thanks!

**Comment by aeneasr** [comment]
Thank you for the changes! Sorry, I'm on vacation right now so reviews take a bit longer :)

**Pull request merged**